Tests/Creation/Frame/Content/String/ID3ChapterTypeFrameCreatorTest.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ID3ChapterTypeFrameCreatorTest.swift
+//  ID3TagEditor
+//
+//  Created by Nolaine Crusher on 4/6/20.
+//  Copyright © 2020 Fabrizio Duroni. All rights reserved.
+//
+
+import XCTest
+@testable import ID3TagEditor
+
+class ID3ChapterTypeFrameCreatorTest: XCTestCase {
+    func testCreatorFrameChapterType() {
+        let id3ChapterTypeFrameCreator = ID3ChapterTypeFrameCreator(
+            frameContentSizeCalculator: ID3FrameContentSizeCalculator(
+                uInt32ToByteArrayAdapter: UInt32ToByteArrayAdapterUsingUnsafePointer(),
+                synchsafeEncoder: SynchsafeIntegerEncoder()
+            ),
+            frameFlagsCreator: ID3FrameFlagsCreator(),
+            stringToBytesAdapter: ID3ISO88591StringToByteAdapter(
+                paddingAdder: PaddingAdderToEndOfContentUsingNullChar(),
+                frameConfiguration: ID3FrameConfiguration()
+            ), frameCreatorsChain: ID3FrameCreatorsChain()
+        )
+        
+        let frameBytes = id3ChapterTypeFrameCreator.createFrame(
+            frameIdentifier: [0x22],
+            version: .version3,
+            elementID: "ch01",
+            startTime: 5000,
+            endTime: 10000,
+            startingByteOffset: 0,
+            endingByteOffset: 0,
+            embeddedSubframes: [.Title : ID3FrameWithStringContent(content:  "Chapter 01")]
+        )
+        
+//        let id3FrameFromStringContentCreator = ID3FrameFromStringContentCreator(
+//            frameContentSizeCalculator: ID3FrameContentSizeCalculator(
+//                uInt32ToByteArrayAdapter: UInt32ToByteArrayAdapterUsingUnsafePointer(),
+//                synchsafeEncoder: SynchsafeIntegerEncoder()
+//            ),
+//            frameFlagsCreator: ID3FrameFlagsCreator(),
+//            stringToBytesAdapter: ID3UTF16StringToByteAdapter(
+//                paddingAdder: PaddingAdderToEndOfContentUsingNullChar(),
+//                frameConfiguration: ID3FrameConfiguration()
+//            )
+//        )
+//        let subframeBytes = id3FrameFromStringContentCreator.createFrame(
+//            frameIdentifier: [0x22],
+//            version: .version3,
+//            content: "Chapter 01")
+        // print(subframeBytes)
+
+        
+        XCTAssertEqual(
+            frameBytes,
+            [
```

Comment:
According to the test, nothing after the `endByteOffset` is being generated. It could be that I set things up wrong in the frame creator, but I'm a little stumped as to how to set them up correctly.

Because we don't know what frame may be generated as a subframe (admittedly, it's almost always just going to be a `.Title` frame but it could technically be ANY frame) I tried to run through the frame creator chain but clearly that's not right.